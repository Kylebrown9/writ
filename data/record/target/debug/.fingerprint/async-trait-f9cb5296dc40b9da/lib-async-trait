6d01f55fe05eda3d