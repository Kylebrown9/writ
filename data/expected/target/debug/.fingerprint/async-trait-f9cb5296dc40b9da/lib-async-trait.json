{"rustc":6064785002308032717,"features":"[]","target":7898274425466113371,"profile":12637318739757120569,"path":8752556564171211336,"deps":[[545221339080631066,"build_script_build",false,3683263555163254387],[2346446859411652336,"syn",false,11430810127383305981],[11187512571400337516,"proc_macro2",false,9475672287560845758],[13094229560132741661,"quote",false,11280425365966564986]],"local":[{"CheckDepInfo":{"dep_info":"debug/.fingerprint/async-trait-f9cb5296dc40b9da/dep-lib-async-trait"}}],"rustflags":[],"metadata":2009543531008976451,"config":2202906307356721367,"compile_kind":0}